shared/common-adapters/name-with-icon.js

```diff
@@ -82,7 +82,7 @@ const NameWithIconVertical = (props: Props) => {
         )}
         {!!props.username && (
           <ConnectedUsernames
-            clickable={true}
+            onUsernameClicked="tracker"
```

Comment:
can we leave it with the previous behavior for now until this component passes through the prop?